# the unique id of this package
uint32 id

# the id of the package type/configuration
uint32 type_id

